pywbem/cim_operations.py

```diff
@@ -1506,6 +1518,16 @@ def operation_recorder_stage_result(self, ret, exc):
             recorder.stage_pywbem_result(ret, exc)
             recorder.record_staged()
 
+    @classmethod
+    def future_logging_reset(cls):
```

Comment:
It seems this function is so out of place compared to the rest of the log config functions, that we are probably better off supporting the reset via the connection=False on configure_logger()